datafusion/substrait/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -607,6 +608,15 @@ async fn qualified_catalog_schema_table_reference() -> Result<()> {
     roundtrip("SELECT a,b,c,d,e FROM datafusion.public.data;").await
 }
 
+#[tokio::test]
+async fn roundtrip_local_relation() -> Result<()> {
+    let ctx = create_context().await?;
+    let row1 = vec![lit(1), lit("a")];
+    let row2 = vec![lit(2), lit("b")];
+    let plan = LogicalPlanBuilder::values(vec![row1, row2])?.build()?;
```

Comment:
I didn't find `VALUES` in the DataFusion SQL - is it not implemented yet?

Reply:
It is implemented, and you can use it like this.
```rust
roundtrip("VALUES (1, 'a', [[1,2,3], []], STRUCT(true, 1))").await
```